engine/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.5.2
 	github.com/bitcoin-sv/go-broadcast-client v0.17.2
 	github.com/bitcoin-sv/go-paymail v0.14.0
+	github.com/bitcoin-sv/spv-wallet/models v0.25.0
```

Comment:
Why we have dependency to models from engine? It shouldn't be here, as models is used only at the API layer